### PR TITLE
Extracted Sale Comments UI from Coin Machine

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Extracted Chat and ChatMessage component from Coin Machine without the logic required to submit a comment

Resolves DEV-249

### CODE REVIEW

**Comment by rdig** [comment]
@ArmandoGraterol this needs to be merged into #2407 's branch `feature/coin-machine-redux`, can you please change the branch before I review this

**Comment by ArmandoGraterol** [comment]
> @ArmandoGraterol this needs to be merged into #2407 's branch `feature/coin-machine-redux`, can you please change the branch before I review this

Done!

**Review by rdig — CHANGES_REQUESTED** [review]
This has a lot of problems. You basically copy-pasted what was in the old #2206 PR which was not what the point of this issue was.

The "extract coin machine" issues have to bring in the new components but in the **context of the actual dapp**.

The old PR was built way before any of the Simplified interface or UAC ever existed. But now, that we'be built them, we have a different context in which to bring in Coin Machine.

So these issues are more of deciding what needs to come in, what needs to be reused, or what needs to be discarded.

Also, the first thing we should do on the Coin Machine front is to bring in the Coin Machine home page grid, in which all these components are aligned into.

This is kind of a bad PR again, but this might not be entirely your fault, since you may have not know to much about the Coin Machine feature.

Next time you try to tackle something like this **please ask for more information** and I'll gladly get you up to speed on what's needed.

As it stands the time you spent on this is kinda wasted, as it does not bring us forward to anything remotely usefull.

Since Coin Machine UI has been put in the back in the backlog until I finish work on Extension versioning, please finish this PR _(so that your work is not wasted)_, but after, when you'll start working on these issues again, please ask me beforehand and I can give you a brief rundown of what's needed on the Coin Machine side.

**Comment by rdig** [comment]
This you push the changes here, can I re-review ?

**Comment by ArmandoGraterol** [comment]
> This you push the changes here, can I re-review ?

Haven't pushed the changes yet, I'm testing them! Should be pushing them soon

**Comment by ArmandoGraterol** [comment]
> This you push the changes here, can I re-review ?

Done! It's ready for a re-review

File: src/modules/core/components/Comment/Comment.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ const handleKeyboardSubmit = (
   return false;
 };
 
-const ActionsPageComment = ({ transactionHash, colonyAddress }: Props) => {
+const Comment = ({ transactionHash, colonyAddress }: Props) => {
```
Comment (rdig) [inline src/modules/core/components/Comment/Comment.tsx:83]: This should be named either `CommentInput` or `CommentBox`

File: src/modules/core/components/FeedItem/FeedItem.tsx [hunk 2]
```diff
@@ -9,11 +9,11 @@ import TextDecorator from '~lib/TextDecorator';
...
 import FriendlyName from '~core/FriendlyName';
 import { useDataFetcher } from '~utils/hooks';
-import { ipfsDataFetcher } from '../../../../core/fetchers';
+import { ipfsDataFetcher } from '../../fetchers';
 
-import styles from './ActionsPageFeedItem.css';
+import styles from './FeedItem.css';
 
-const displayName = 'dashboard.ActionsPageFeed.ActionsPageFeedItem';
+const displayName = 'FeedItem';
```
Comment (rdig) [inline src/modules/core/components/FeedItem/FeedItem.tsx:16]: This should be renamed to `Comment` or `CommentItem` (as the feed name works only in the context of the Actions Page)

**Review by rdig — APPROVED** [review]
I'm ok with this, but please make sure to rename the core comment components.

I've left comments on what names you should choose.

After that, you can merge it.

**Pull request merged**